apps/services/auth/delegation-api/test/access-test-cases.ts

```diff
@@ -302,4 +318,80 @@ export const accessTestCases: Record<string, TestCase> = {
     ],
     expected: [],
   },
+  // Should get list of all domains if not filtered by direction.
+  allDomains: {
```

Comment:
Arguably these test cases should only focus on outgoing filtering. We can create separate test cases or stand-alone tests for this. Otherwise we're running a lot of invalid tests on this test-cases.